modules/http-helper/http_helper.go

```diff
@@ -16,10 +16,25 @@ import (
 	"github.com/gruntwork-io/terratest/modules/testing"
 )
 
+type HttpGetOptions struct {
+	Url       string
+	TlsConfig *tls.Config
+	Timeout   int
+}
+
+type HttpDoOptions struct {
+	Method    string
+	Url       string
+	Body      []byte
+	Headers   map[string]string
+	TlsConfig *tls.Config
+	Timeout   int
+}
+
 // HttpGet performs an HTTP GET, with an optional pointer to a custom TLS configuration, on the given URL and
 // return the HTTP status code and body. If there's any error, fail the test.
-func HttpGet(t testing.TestingT, url string, tlsConfig *tls.Config) (int, string) {
-	statusCode, body, err := HttpGetE(t, url, tlsConfig)
+func HttpGet(t testing.TestingT, options *HttpGetOptions) (int, string) {
```

Comment:
I really like the `Options` struct approach, but these sorts of breaking changes are quite painful. Is it worth adding new methods that take in the `Options` struct and initially deprecate the old methods?

Reply:
Spending some time thinking through this

Reply:
@brikis98 #1078 uses your suggested approach and adds new methods that accept the options structs